src/usr/local/www/services_captiveportal.php

```diff
@@ -633,13 +651,19 @@ function build_authserver_list() {
 	))->setHelp("If enabled, connected users won't be disconnected during a pfSense reboot.");
 }
 
-$section->addInput(new Form_Checkbox(
+$concurrentlogins_list = array("disabled" => "Disabled", "multiple" => "Multiple", "last" => "Last login", "first" => "First login");
+
+$section->addInput(new Form_Select(
 	'noconcurrentlogins',
 	'Concurrent user logins',
-	'Disable Concurrent user logins',
-	$pconfig['noconcurrentlogins']
-))->setHelp('If enabled only the most recent login per username will be active. Subsequent logins will cause machines previously logged in with the ' .
-			'same username to be disconnected.');
+	$pconfig['noconcurrentlogins'],
+	$concurrentlogins_list
+))->setHelp('Disabled : do not allow concurrent logins per username or voucher.<br />'.
+			'Multiple : no restrictions to the number of logins per username or voucher will be applied.<br />'.
+			' Last login : only the most recent login per username or voucher will be granted.'.
+			' Previous logins will be disconnected.<br />'.
```

Comment:
Put the `<br />` parts in separate strings so they don't get put into gettext strings. Or use the sprintf style. There are various examples on other pages.